string task_id
string robot_id
---
bool success
